_posts/2016-11-09-making-a-small-object.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: Making a Small Object
+excerpt: The thought process behind creating a simple object to help deal with complex ActiveRecord ORDER BY queries with a strong focus on the Single Responsibility Principal.
```

Comment:
This is not a comment you necessarily have to act upon, but it's often helpful to figure out up front your primary motivation for writing this post. Is your primary motivation showing off architectural / design ability (i.e., breaking down unwieldy large into easy-to-use small things) or deeper technical knowledge in ActiveRecord / relational databases? Obviously the post can show evidence of both, but you will want to spend more of your word budget on your primary motivation. Based on your answer to that question, you can place less weight on my comments below which pertain to your secondary motivation.